website/docs/metrics.md

```diff
@@ -108,7 +110,7 @@ Below are the list of metrics provided by Gatekeeper:
 
     Aggregation: `LastValue`
 
-- Name: `gatekeeper_constraint_template_ingestion_count`
+- Name: `gatekeeper_constraint_template_ingestion_count_total`
```

Comment:
Since the _total is appended by client libraries, I think we should keep the original ending in _count since that is what we have in the code and user might not use prometheus. For ease of search and lookup, we could add the _total one next to the existing name. wdyt?